pages/content/amp-dev/documentation/courses/beginning-course/thinking-in-components@pt_BR.md

```diff
@@ -0,0 +1,265 @@
+
+{{ image('/static/img/courses/beginner/image23.png', 311, 550,  align='center third', caption='A imagem da bicicleta sai da borda da tela') }}
+
+A imagem que adicionamos à página não encolhe para caber em telas menores; apenas derrama para o lado. Se não especificarmos uma estratégia para dispor a imagem e redimensioná-la, o padrão será uma largura e altura fixas, conforme especificamos em nosso código. Felizmente, podemos corrigir esse problema usando o sistema de layout de AMP.
+
+Vamos dar à nossa imagem um `layout` do tipo` responsive` para que ela seja dimensionada automaticamente conforme a janela é redimensionada. O layout responsivo faz com que a imagem assuma as dimensões do contêiner pai, ao mesmo tempo que respeita a proporção original. Se o contêiner pai tiver apenas 320 pixels de largura, a imagem manterá sua proporção e será redimensionada para 320 x 240 (em vez de 640 x 480).
+
+Adicione o atributo `layout` à nossa imagem. Se feito corretamente, será semelhante a este:
+
+[sourcecode:html]
+{% raw %}<amp-img src="https://cdn.glitch.com/d7f46a57-0ca4-4cca-ab0f-69068dec6631%2Fricotta-racer.jpg?1540228217746" layout="responsive" width="640" height="480"></amp-img>
+{% endraw %}[/sourcecode]
+
+Depois de fazer a alteração, dê uma olhada em sua página. A imagem tem a proporção correta e preenche a largura da tela de maneira responsiva. Problema resolvido.
+
+{{ image('/static/img/courses/beginner/image26.png', 311, 550,  align='center third', caption='Imagem de bicicleta com proporção correta') }}
+
+Existem outros tipos de layout além do responsivo (pelo menos 8 no total, na verdade).
+Por exemplo, o layout `fixed` indica que o componente nunca deve ser redimensionado da altura e largura atribuídas a ele. O layout `intrinsic` é semelhante ao layout `responsive`, exceto que tem o conceito de altura e largura intrínsecas que não podem ser excedidas pelo componente. Alguns layouts só podem ser aplicados a determinados componentes. A documentação de cada componente especificará quais layouts são válidos para aquele componente. Você pode ler sobre o resto dos tipos de layout [aqui](../../../documentation/guides-and-tutorials/learn/amp-html-layout/layouts_demonstrated.html).
+
+Se você deseja se tornar um desenvolvedor de AMP de sucesso, aprender a usar o sistema de layout é fundamental. Todos os layouts fornecidos pelo AMP podem ser implementados usando CSS simples, mas geralmente podem ser complicados ou ter casos extremos complicados que exigem conhecimento profundo para serem solucionados. O AMP simplifica o processo e expõe muitas dessas opções de layout para serem usadas em qualquer elemento da página AMP. Confira o
+[documentação oficial](../../../documentation/guides-and-tutorials/learn/amp-html-layout/index.md) para obter mais informações sobre o sistema de layout.
```

Comment:
same as the following comment related to the locale path

Reply:
Done!